### ISSUE TITLE
1275 Error

### ISSUE THREAD

**Issue by jturner900** [MartinDrab/IRPMon#7]
Trying to use the IRPMon.exe executable in Windows 10 and am getting a 1275 error.

"Failed to start the IRPMon driver
 Error code: 1275
 Error message: This driver has been blocked from loading"

Also get this box:

" A digitally signed driver is required: irpmndrv.sys

Windows blocked the installation of a digitally unsigned driver. Uninstall the program or device that uses the driver and check the publisher's website for a digitally signed version of the driver."

Do machines normally apply resistance to this application?

**Comment by MartinDrab** [comment]
This message usually appears when you try to run 32-bit binaries (drivers) on 64-bit WIndows version. 

Also, the drivers are not digitally signed, so the system needs to be set accordingly (otherwise, you usually run into the error 577).

**Comment by jturner900** [comment]
Thanks Martin.  Just one more question.  Is there a general guide for how to add devices and classes?  I'm choosing to add Keyboard and Mouse classes, enabling logging, and am getting nothing under Requests.

**Comment by MartinDrab** [comment]
Adding IRPMon as a filter for a device class hay have effects only on newly connected devices of that class (connected after you add IRPMon as a filter). If you need to filter existing devices of that class, you need to know them and selected them manually (\Device\KeyboardClassX for keyboards, \Device\PointerClassX for mice).

Unfortunately, the current documentation for IRPMon does not cover the GUI application, as far as I remember. I am planning to improve this in quite near future.

**Comment by MartinDrab** [comment]
The documentation actually contains instruction how to monitor manually selected drivers and devices. I have also added some description related to monitoring devices and drivers not currently present in the system (PnP ones). I have included the latest version of the documentation in the binaries folder.

After the first oficial release (which should happen within few days), I plan to write a blogpost about monitoring keyboard devices with IRPMon which should be exactly what you are looking for.
